Update Readme for Quickstart (#14)
codalab/worksheets-examples: 00-quickstart/README.md

> ## The tutorial could be found here: [CodaLab Worksheets Documentation](https://codalab-worksheets.readthedocs.io/en/latest/examples/quickstart/)

Reviewer: ```suggestion
This tutorial can be found here: [CodaLab Worksheets Documentation](https://codalab-worksheets.readthedocs.io/en/latest/examples/quickstart/)
```
